### PR TITLE
Add contributing.md

### LINKED ISSUE #733: Getting started guide
How to get started contributing to the project. Starting watch mode and adding the extension as development extension to Chrome, etc. See: https://github.com/sindresorhus/refined-github/issues/688#issuecomment-337463374

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #733

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -192,46 +192,7 @@ While this project is highly opinionated, this doesn't necessarily limit you fro
...
-			<ol>
-				<li>Open <code>about:debugging#addons</code>
-				<li>Click on the <strong>Load Temporary Add-on</strong> button
-				<li>Select the file <code>refined-github/distribution/manifest.json</code>
-			</ol>
-		</td>
-	</tr>
-</table>
-
+See [`CONTRIBUTING.md`](CONTRIBUTING.md)
```
Comment (sindresorhus) [inline readme.md]: > See the [contribution guide](CONTRIBUTING.md).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+Suggestions and pull requests are highly encouraged! Have a look at the [open issues](https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+sort%3Areactions-%2B1-desc), especially [the easy ones](https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+sort%3Areactions-%2B1-desc).
+
+## Notions
+
+- You will need to be familiar with [npm](https://docs.npmjs.com/getting-started/) and [webpack](https://web-design-weekly.com/2014/09/24/diving-webpack/) to build this extension.
+- The extension can be loaded into Chrome or Firefox manually ([See notes below](#loading-into-the-browser))
+- [JSX](https://reactjs.org/docs/introducing-jsx.html) is used to create dom elements
```
Comment (sindresorhus) [inline CONTRIBUTING.md]: `dom` => `DOM`

**Comment by sindresorhus** [comment]
I think it should be lowercase and put in a `.github` directory. There's no good reason to use uppercase in these kinds of files other than habit or legacy.

**Comment by bfred-it** [comment]
I don't think so, `.github` files are just for github, but `/contributing.md` would be seen by everyone. 

GitHub also picks it up automatically, for example on the [rust repo](https://github.com/rust-lang/rust)

contributing banner

**Comment by sindresorhus** [comment]
I don't feel strongly about putting it in `.github`, but since we already link to it from the readme and GitHub picks it up too, it seems visible enough.

**Comment by bfred-it** [comment]
Cool, then! :) The other two changes were made

**Pull request merged**